README.md

```diff
@@ -1,20 +1,25 @@
-# MODULE
+# MODULE_NAME Service
 
-![Rust Checks](https://github.com/arrow-air/svc-template-rust/actions/workflows/rust.yml/badge.svg?branch=${GH_BRANCH}) (Replace Repository Name!)
+![Rust Checks](https://github.com/arrow-air/svc-template-rust/actions/workflows/rust_ci.yml/badge.svg?branch=main)
+![Python Flake8](https://github.com/arrow-air/svc-template-rust/actions/workflows/python_ci.yml/badge.svg?branch=main)
+![CodeClimate Coverage](https://img.shields.io/codeclimate/coverage/arrow-air/svc-template-rust)
 
-![Python Flake8](https://github.com/arrow-air/svc-template-rust/actions/workflows/python.yml/badge.svg?branch=${GH_BRANCH}) (Replace Repository Name!)
+FIXME: Replace the repository name for each:
 
-![CodeClimate Coverage](https://img.shields.io/codeclimate/coverage/arrow-air/svc-template-rust) (Replace Repository Name!)
+## :telescope: Overview
 
-## Overview
+*FIXME: This is a high level description of this module.*
 
-*This is a high level description of this module.*
-
-## Documentation
+## :scroll: Documentation
 
 - [Concept of Operations](FIXME)
 - [Requirements & User Stories](FIXME)
 - [SDD](./docs/sdd.md)
 
+## :busts_in_silhouette: Arrow DAO
+
+- [Website](https://www.arrowair.com/)
+- [Arrow Docs](https://www.arrowair.com/docs/intro)
+- [Discord](https://discord.com/invite/fab4bxaAW9)
```

Comment:
https://discord.com/invite/arrow

Reply:
Fixed in https://github.com/Arrow-air/svc-template-rust/pull/3/commits/003d7fde107fde2416c259c00b406c505e6093ab